lib/SILOptimizer/Mandatory/PerformanceDiagnostics.cpp

```diff
@@ -353,7 +353,26 @@ bool PerformanceDiagnostics::visitInst(SILInstruction *inst,
   SILType impactType;
   RuntimeEffect impact = getRuntimeEffect(inst, impactType);
   LocWithParent loc(inst->getLoc().getSourceLoc(), parentLoc);
-  
+
```

Comment:
It would be better to define an "Existential" runtime effect here https://github.com/apple/swift/blob/main/include/swift/SIL/RuntimeEffect.h and add those instructions in  `swift::getRuntimeEffect` https://github.com/apple/swift/blob/99a36b3f00cc98c11e4bb674c9ea743d448ac0d2/lib/SIL/Utils/InstructionUtils.cpp#L438